api/v1/editTaskTemplates/resources/EditTaskTemplateResource.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file api/v1/editTaskTemplates/resources/EditTaskTemplateResource.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2003-2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class EditTaskTemplateResource
+ *
+ * @brief Transforms an editorial task template into an API response format.
+ *
+ */
+
+namespace PKP\API\v1\editTaskTemplates\resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class EditTaskTemplateResource extends JsonResource
```

Comment:
This is already added in the main: https://github.com/pkp/pkp-lib/blob/main/api/v1/editTaskTemplates/resources/TaskTemplateResource.php